contracts/interfaces/compound/ICToken.sol

```diff
@@ -2,28 +2,66 @@
 
 pragma solidity =0.8.9;
 
+import "./InterestRateModel.sol";
+
 // Subset of https://github.com/compound-finance/compound-protocol/blob/master/contracts/CTokenInterfaces.sol
-interface ICToken {
+contract CTokenStorage {
```

Comment:
Because we use storage now we can't have this as an interface. The Borrow Oracle calls getter functions (e.g. https://github.com/Voltz-Protocol/voltz-core/blob/ir/compound-borrow-rate-oracle/contracts/rate_oracles/CompoundBorrowRateOracle.sol#L46). We can rename this to "CTokenInterface" as Compound did to avoid confusion with an actual interface... what do you think?

Reply:
I removed the storage contract and added getters in the `ICToken` definition. We can make it an interface now to follow best practices.  The `borrowRateMaxMantissa` is now defined in oracle itself to be used in the require statement `require(borrowRateMantissa <= borrowRateMaxMantissa,"borrow rate is absurdly high");`